blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
+
+.table-form-submit-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-form-submit-wrapper button {
+    height: 50px;
+    width: 300px;
+    background-color: var(--blue);
+    color: var(--white);
+    border-radius: 6px;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-s);
+    margin: 20px 0;
+}
+
+.table-form-submit-wrapper button:disabled {
+    background-color: var(--blue-light);
+}
+
+.table-wrapper:not(:has(+ .table-wrapper)) .table tbody tr:last-of-type td {
+    border-bottom: 2px solid var(--blue);
+}
+
+.table {
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+}
+
+.table:first-of-type {
+    border-top: 2px solid var(--blue);
+}
+
+.table table {
+    border-spacing: 0;
+    width: 100%;
+}
+
+.table thead th {
+    padding: 0 12px;
+    border-bottom: 2px solid var(--blue);
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xs);
+    text-transform: uppercase;
```

Comment:
these seem to have no effect (overridden by other css). necessary?

Reply:
Can you clarify? I'm seeing changes to the header text when it's removed.

I see this before: ![Screen Shot 2024-10-01 at 9 36 35 AM](https://github.com/user-attachments/assets/2964497f-0647-4268-a1f6-a29400a79196)

and this after: 
![Screen Shot 2024-10-01 at 9 36 49 AM](https://github.com/user-attachments/assets/7067e98f-a096-4167-9734-634b7c770608)

Reply:
Never mind, I found the problem you're seeing. I'll get that fixed :)